### ISSUE TITLE
No Windows Installation: install is absent, VS is not available 

### ISSUE THREAD

**Issue by chang-zhao** [zephir-lang/zephir#1797]
I'm trying to install Zephir on Windows 7 according to the docs:

https://docs.zephir-lang.com/0.11/en/installation

```
$ git clone https://github.com/phalcon/zephir
$ cd zephir
$ ./install -c
```
Then I get `bash: ./install: No such file or directory`

Then I try to follow WINDOWS.md from zephir folder, it says:

```
- [Install Visual Studio 2012 Express](http://www.microsoft.com/en-US/download/details.aspx?id=34673)
```
That link says: "We're sorry, this download is no longer available."

**Comment by danhunsaker** [comment]
The docs need updating, for sure. Zephir is now installed using Composer, or by grabbing the latest PHAR from GitHub.

Note, however, that Windows 7 may not be supported properly.

**Comment by chang-zhao** [comment]
Make a checklist as a part of the development process. Like,
```
333. Create the release.
334. Update docs...
```
For now, I would like to know exactly:

  * Is Wndows 7 supported properly?
  * Or should I use Linux?

Otherwise I shouldn't waste my time.

Thank you.

**Comment by danhunsaker** [comment]
How do I put this without being rude ....

Windows 7 is very old, and isn't supported by _most_ software released in the last few years, including PHP itself. It will stop receiving security updates in less than a year (though, to be fair, only _just_ less). It's a good idea to move to something else, be it Windows 10, Linux, or something else entirely, just in general, not just for Zephir support.

Will it work for now? Most likely. But you're on your own if it doesn't, just as you would be for any other project.

**Comment by chang-zhao** [comment]
That is not so. Windows 7 is being supported by the latest PHP.
https://secure.php.net/manual/en/install.windows.requirements.php
Anyway, with your attitude Zephir seems not worth trying.
Goodbye.

**Comment by danhunsaker** [comment]
I stand corrected on PHP's support status, though I'll point out that Windows has always been a second-class citizen with PHP support, no matter the version.

That said, it's still dangerously close to EOL, and the support overhead is excessive for a project with a team this small. So we won't be able to help much with Win7-specific issues. That's my entire point, there.

I agree, all that said, that my responses have been antagonistic, and I'll need to address that.

**Comment by chang-zhao** [comment]
People might try the software on their desktop OS, and then recompile it on Linux for production. So there is no issue, Windows 7 or not.

As for support, I have no idea what special support it would need.

Anyway, the documentation is messed up, the installation procedure is messed up. First I had to installl MinGW, then it appears it doesn't have the necessary commands, so I had to install MSYS, then it was 32-bit and something not worked, so I had to reinstall MSYS2 with its own connection to MinGW. All that required hours of surfing the internet, as search engines didn't give any ready step-by-step up-to-date instruction. At last, I managed to install re2c and the Zephir parser. Then I had to install VS - who knows which: 2012, 2015, 2017..? I went for 2017, as I need PHP7... I even tried Composer, but when it said my PHP doesnt have GMP extension, and I found no ready dll for it, that was it for me. I can imagine that compiling (first time in my life) a PHP extension for Windows would be just another step in who knows how long procedure of getting Zephir, but I give up.

With such docs, installation procedures, developers attitude - I decided Zephir, even though looking super interesting, might be just a huge waste of time with unclear benefits. I better put my efforts in actual development of my software, like before, for now, putting such experiments aside.
